mingw-w64-dlfcn/PKGBUILD

```
@@ -3,47 +3,62 @@
 _realname=dlfcn
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=1.2.0
-pkgrel=2
+pkgver=1.3.0
+pkgrel=1
 pkgdesc="A wrapper for dlfcn to the Win32 API (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 url="https://github.com/dlfcn-win32/dlfcn-win32"
-license=('LGPL')
+license=('MIT')
 depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs")
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-cmake")
 options=('strip' 'staticlibs')
 source=(${_realname}-${pkgver}.tar.gz::https://github.com/dlfcn-win32/dlfcn-win32/archive/v${pkgver}.tar.gz)
-sha256sums=('f18a412e84d8b701e61a78252411fe8c72587f52417c1ef21ca93604de1b9c55')
+sha256sums=('24c69d43ddc9243fd2639a07495a7e0714278e8d3d0e124afdbab892dbb4a92d')
 
 prepare() {
   cd "${srcdir}/dlfcn-win32-${pkgver}"
 }
 
 build() {
   for SHARED in YES NO; do
-    [[ -d "${srcdir}/build-${MINGW_CHOST}-${SHARED}" ]] && rm -rf "${srcdir}/build-${MINGW_CHOST}-${SHARED}"
-    cp -rf "$srcdir/dlfcn-win32-${pkgver}" "${srcdir}/build-${MINGW_CHOST}-${SHARED}"
-    cd "${srcdir}/build-${MINGW_CHOST}-${SHARED}"
+    [[ -d "${srcdir}/build-${MSYSTEM}-${SHARED}" ]] && rm -rf "${srcdir}/build-${MSYSTEM}-${SHARED}"
+    cp -rf "$srcdir/dlfcn-win32-${pkgver}" "${srcdir}/build-${MSYSTEM}-${SHARED}"
+    cd "${srcdir}/build-${MSYSTEM}-${SHARED}"
     MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
       ${MINGW_PREFIX}/bin/cmake \
         -G "MSYS Makefiles" \
         -DBUILD_SHARED_LIBS=${SHARED} \
         -DCMAKE_INSTALL_PREFIX="${MINGW_PREFIX}"  \
+        -DBUILD_TESTS=ON \
         .
     make
   done
 }
 
+check() {
+ for SHARED in YES NO; do
+   msg "Test: Mingw host:       ${MINGW_CHOST}"
+   msg "Test: MSYSTEM variable: ${MSYSTEM}"
+   msg "Test: Shared version:   ${SHARED}"
+   cd "${srcdir}/build-${MSYSTEM}-${SHARED}"
+    make test
+  done
+}
+
 package() {
   for SHARED in YES NO; do
-    cd "${srcdir}/build-${MINGW_CHOST}-${SHARED}"
+    cd "${srcdir}/build-${MSYSTEM}-${SHARED}"
     make DESTDIR="${pkgdir}" install
   done
-  # We do not want transitive dependencies on libdll.dll so remove it.
+  cd ${srcdir}/dlfcn-win32-${pkgver}
+  install -Dm644 README.md "${pkgdir}${MINGW_PREFIX}/share/doc/${_realname}/README.md"
+  install -Dm644 COPYING "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE"
+  # We do not want transitive dependencies on libdll.dll so remove it
+  # as well as a test executable.
   # We installed static last so the CMkake files pertain to a static
```

you could fix this typo
```suggestion
  # We installed static last so the CMake files pertain to a static
```